packages/router/src/Exceptions/ControllerMethodHasMultipleRoutes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Router\Exceptions;
+
+use Exception;
+use Tempest\Support\Arr;
+
+final class ControllerMethodHasMultipleRoutes extends Exception implements RouterException
+{
+    /** @param string[] $routes */
+    public function __construct(string $controllerClass, string $controllerMethod, array $routes)
```

Comment:
We don't need an exception here. We just need to use the first route for URI generation.